transcriptclean.wdl

```diff
@@ -0,0 +1,199 @@
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task transcriptclean {
+    input {
+        File SAMfile
+        File referenceFile
+        String outputPrefix
+        String outputDirPath
+        File? spliceJnsFile
+        File? variantsFile
+        Int? maxLenIndel
+        Int? maxSJoffset
+        Boolean? correctMismatches
+        Boolean? correctIndels
+        Boolean? correctSJs
+        Boolean? dryRun
+        Boolean? primaryOnly
+        Int cores = 1
+        Int memory = 25
+        String dockerImage = "biocontainers/transcriptclean:v1.0.7_cv1"
+    }
+
+    command {
+        set -e pipefail
+        mkdir -p ~{outputDirPath}
+        TranscriptClean \
+            ~{"-s " + SAMfile} \
+            ~{"-g " + referenceFile} \
+            ~{"-o " + outputDirPath + outputPrefix} \
+            ~{"-j " + spliceJnsFile} \
+            ~{"-v " + variantsFile} \
+            ~{"--maxLenIndel=" + maxLenIndel} \
+            ~{"--maxSJOffset=" + maxSJoffset} \
+            ~{true="-m CORRECTMISMATCHES" false="-m false" correctMismatches} \
+            ~{true="-i CORRECTINDELS" false="-i false" correctIndels} \
+            ~{true="--correctSJs=CORRECTSJS" false="--correctSJs=false" correctSJs} \
+            ~{true="--dryRun" false="" dryRun} \
+            ~{true="--primaryOnly" false="" primaryOnly}
+    }
+
+    output {
+        File outputTCfasta = outputDirPath + outputPrefix + "_clean.fa"
+        File outputTClog = outputDirPath + outputPrefix + "_clean.log"
+        File outputTCsam = outputDirPath + outputPrefix + "_clean.sam"
+        File outputTCteLog = outputDirPath + outputPrefix + "_clean.TE.log"
+    }
+
+    runtime {
+        cpu: cores
+        memory: memory
+        docker: dockerImage
+    }
+}
+
+task cleansplicejns {
+    input{
+        File SAMfile
+        File referenceFile
+        String outputPrefix
+        String outputDirPath
+        File spliceJNsFile
+        File? variantsFile
+        Int cores = 1
+        Int memory = 4
+        String dockerImage = "biocontainers/transcriptclean:v1.0.7_cv1"
+    }
+
+    command {
+        set -e pipefail
+        mkdir -p ~{outputDirPath}
+        clean_splice_jns \
+            ~{"--f=" + SAMfile} \
```

Comment:
Why indented?

Reply:
The arguments I indented are additions to the clean_splice_jns call, but not to the other bash calls (like mkdir) in the command section. So I thought the indents would make that separation more transparent.
I have removed the extra indents from the command sections.